packages/legend-query/src/components/__tests__/QueryBuilder_FetchStructure.test.tsx

```diff
@@ -370,6 +370,10 @@ test(
     const _personClass = mockedQueryStore.graphManagerState.graph.getClass(
       'model::pure::tests::model::simple::Person',
     );
+    const _personExtensionClass =
```

Comment:
So after discussing with @MauricioUyaguari I think mapped-ness algorithm is getting more and more heavy, which demands its own test file. Let's create a new test file and do **unit-testing** just the `mapped-ness algorithm`.